Document authentication settings. (#69284)
elastic/kibana: docs/user/security/authentication/index.asciidoc

Enable multiple authentication mechanisms at the same time specifying a prioritized list of the authentication _providers_ (typically of various types) in the configuration. Providers are consulted in ascending order. Make sure each configured provider has a unique name (e.g. `basic1` or `saml1` in the configuration example) and `order` setting. In the event that two or more providers have the same name or `order`, {kib} will fail to start.

When two or more providers are configured, you can choose the provider you want to use on the Login Selector UI. The order the providers appear is determined by the `order` setting. The appearance of the specific provider entry can be customized with the `description`, `hint` and `icon` settings.

> TIP: You can also provide users with a more generic guidance on the login process using `xpack.security.loginHelp` setting that supports Markdown format. When you specify this setting Login Selector UI will display a `Need help?` link that will let users to access an additional help information.

Author: __note:__ `a more generic guidance on the login process` sounds .... weird... :see_no_evil: This setting is supposed to explain users what login method to use, maybe provide help/reset-password links etc.

Reviewer: How about:

To provide login instructions to users, use the `xpack.security.loginHelp` setting, which support Markdown.

Author: Sounds good, thanks!

> TIP: You can also provide users with a more generic guidance on the login process using `xpack.security.loginHelp` setting that supports Markdown format. When you specify this setting Login Selector UI will display a `Need help?` link that will let users to access an additional help information.

Reviewer: ```suggestion
TIP: You can also provide users with a more generic guidance on the login process using `xpack.security.loginHelp` setting that supports Markdown format. When you specify the `xpack.security.loginHelp` setting, the Login Selector UI displays a `Need help?` link that lets users access login help information.
```

    enabled: false
--------------------------------------------------------------------------------

[role="screenshot"]

> image::user/security/images/kibana-login.png["Login Selector UI"]

Reviewer 2: nit: given the configuration above, shouldn't I expect to see the Kerberos realm in this screenshot?

Author: Nope it's `enabled: false` :slightly_smiling_face:

Author: I kept it to show that `enabled` also affects selector (obviously though), but I can remove if it's more confusing than helpful?

Reviewer 2: Ah I missed that!

> For more information, see <<authentication-security-settings, authentication security settings>>.

Reviewer: ```suggestion
For more information, refer to <<authentication-security-settings, authentication security settings>>.
```
